case-study-hydropro-solutions.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: 'HydroPro Solutions achieves a 60% reduction in search time using the Xata Postgres-powered platform'
+description: 'Discover how HydroPro Solutions reduced search time by 60% and transformed their operations with the powerful Postgres-based serverless database solution from Xata, streamlining water meter installations and improving overall efficiency.'
+image:
+  src: https://us-east-1.storage.xata.sh/uq2d57ppt77acjdd5ptct28claqh8c1g
+  alt: Case Study
+author: Cezzaine Zaher
+date: 09-19-2024
+tags: ['case-study', 'community']
+published: true
+slug: case-study-hydropro-solutions
+ogImage: https://us-east-1.storage.xata.sh/uq2d57qfhg4d1ill2ahaevavdrfnd9cs
+---
+
+<ArticleImage
+  src="https://us-east-1.storage.xata.sh/uq2d570nl53n5ssl5pv7p6140l9b6j9k"
+  width="800"
+  height="405"
+  alt="Savas labs hero image"
+  position="center"
+/>
+
+- **Company:** HydroPro Solutions
+- **Industry:** Water utilities
+- **Location:** Texas & California
+- **Size:** 30 employees
+
+[HydroPro Solutions](https://hydroprosolutions.com/) is a leading sales and distribution company providing smart water meters and sewage monitoring systems to small and medium-sized water utilities across Texas and California. With a team of 30 employees, they specialize in advanced water infrastructure services, including pipeline pressure monitoring and cyber threat detection, ensuring municipalities can manage their water resources effectively and securely.
+
+## Challenges and the task ahead
+
+As the sole platform engineer at HydroPro Solutions, Chris Hiester faced a significant challenge: building an entire software platform to manage large-scale water meter replacement projects. These projects involved managing data for up to 20,000 water meters in a single city, including customer information, meter readings, work orders, and high-resolution images of the installations.
+
+Chris needed a database solution that could handle vast amounts of relational data while being easy to manage as a single developer. The system also needed to support search capabilities for quickly locating customer records and photos, avoiding costly billing disputes. Additionally, HydroPro needed the platform to integrate smoothly into their existing tech stack, which included Next.js, Vercel, and Google Cloud Vision.
+
+> We needed a PostgreSQL system that would just work and perform well without the hassle of dealing with core database infrastructure. As a single engineer, I didn’t want to spend time managing configurations—Xata’s APIs and search capabilities made it a perfect fit.\\
```

Comment:
```suggestion
> We needed a PostgreSQL system that would just work and perform well without the hassle of dealing with core database infrastructure. As a single engineer, I didn’t want to spend time managing configurations—Xata’s APIs and search capabilities made it a perfect fit.\
```

I think we only need one backslash, it's showing up in the quote.

![CleanShot 2024-09-19 at 11 15 30@2x](https://github.com/user-attachments/assets/398304a6-f067-46ef-8598-c119bce51fe0)